ui/lib/pki/addon/routes.js

```diff
@@ -3,45 +3,44 @@ import buildRoutes from 'ember-engines/routes';
 export default buildRoutes(function () {
   this.route('overview');
   this.route('configuration', function () {
-    this.route('index', { path: '/' });
     this.route('tidy');
-    this.route('create', function () {
-      this.route('index', { path: '/' });
-      this.route('import-ca');
-      this.route('generate-root');
-      this.route('generate-csr');
-    });
+    this.route('create');
```

Comment:
The initial configuration has three radio buttons that provide a different configuration view, but they are all the same route
<img width="743" alt="Screen Shot 2022-11-16 at 3 34 34 PM" src="https://user-images.githubusercontent.com/68122737/202317805-85c4d937-b850-47aa-aa1b-acba5b38525d.png">